data/siteText.yaml

```diff
@@ -27,15 +27,14 @@ envEffects-oysters2: particles per serving of commercially-cultured <strong>oyst
 envEffects-oysters3: 90
 envEffects-oysters4: "particles per serving of commercially-cultured <strong>mussels</strong><sup><a href='#ref2'>2</a></sup>"
 
-envEffects-stream1: 1 particle for every
-envEffects-stream2: 8
-envEffects-stream3: gallon of Great Lakes tributary <strong>water</strong><sup><a href='#ref3'>3</a></sup>
-envEffects-stream4: "On average"
-envEffects-stream5: 1285
-envEffects-stream6: "particles per square foot of <strong>river sediment</strong><sup><a href='#ref4'>4</a></sup>"
+envEffects-stream1: Up to 
```

Comment:
It looks like you need to do an upstream pull. The changes I made to the references need to be retained too.